ex1/functions.py

```diff
@@ -46,15 +45,91 @@ def f():
 ##############################
 
 # Define a function named add that takes two numbers and returns the sum.
+def add(a,b):
+	"""
+    Adds two parameters and returns its sum
 
+    Parameters
+    ----------
+    a: int or float
+        First summand
+    b: int or float
+        Second summand
+
+    Returns
+    -------
+    a+b: int or float
+        Sum of a and b
+    """
+	return a+b
+	
 # Define a function named to_tuple that takes three arguments and returns a
 # tuple of these three arguments.
 
+def to_tuple(a,b,c):
+	"""
+    Adds there parameters to a tuple
+
+    Parameters
+    ----------
+    a: object
+        First object to add to tuple
+    b: object
+        Second object to add to tuple
+	c: object
+		Third object to add to tuple
+
+    Returns
+    -------
+	(a,b,c)
+        Tuple of a and b and c
+    """
+	return (a,b,c)
```

Comment:
You do not need the brackets here since multiple return values are automatically packed into a tuple.